leroy-merlin-br/php-coding-standard
# Add tests for LMBR PHP Coding Standard

With this PR, I'd like to introduce some tests to our CS repository. Fixes #4 

## How it works?

We have two folders inside `tests/`, one called `fixed/` and another one `input/`. As the names suggest, the files under `input/` are **wrong** files that are gonna be compared with the ones under `fixed/`.

## How to compare the files?

We have a committed file called `expected_report.txt`. It's exactly like the report from PHPCS, and this is how we're gonna test: run the PHPCS and compare its report with this one.

## How to run the tests?

I've created an alias under Composer to wrap the command we should run to run the tests: `composer test`.

Important: **THIS PR DOES NOT INTRODUCE ANY NEW RULE**. So, if you found something strange, probably is because there's no rule for that, and you're free to open a PR to add it :tada:
